specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7837,179 +7837,7 @@
-            "Standard_F8",
-            "Standard_F16",
-            "Standard_F1s",
-            "Standard_F2s",
-            "Standard_F4s",
-            "Standard_F8s",
-            "Standard_F16s",
-            "Standard_F2s_v2",
-            "Standard_F4s_v2",
-            "Standard_F8s_v2",
-            "Standard_F16s_v2",
-            "Standard_F32s_v2",
-            "Standard_F64s_v2",
-            "Standard_F72s_v2",
-            "Standard_G1",
-            "Standard_G2",
-            "Standard_G3",
-            "Standard_G4",
-            "Standard_G5",
-            "Standard_GS1",
-            "Standard_GS2",
-            "Standard_GS3",
-            "Standard_GS4",
-            "Standard_GS5",
-            "Standard_GS4-8",
-            "Standard_GS4-4",
-            "Standard_GS5-16",
-            "Standard_GS5-8",
-            "Standard_H8",
-            "Standard_H16",
-            "Standard_H8m",
-            "Standard_H16m",
-            "Standard_H16r",
-            "Standard_H16mr",
-            "Standard_L4s",
-            "Standard_L8s",
-            "Standard_L16s",
-            "Standard_L32s",
-            "Standard_M64s",
-            "Standard_M64ms",
-            "Standard_M128s",
-            "Standard_M128ms",
-            "Standard_M64-32ms",
-            "Standard_M64-16ms",
-            "Standard_M128-64ms",
-            "Standard_M128-32ms",
-            "Standard_NC6",
-            "Standard_NC12",
-            "Standard_NC24",
-            "Standard_NC24r",
-            "Standard_NC6s_v2",
-            "Standard_NC12s_v2",
-            "Standard_NC24s_v2",
-            "Standard_NC24rs_v2",
-            "Standard_NC6s_v3",
-            "Standard_NC12s_v3",
-            "Standard_NC24s_v3",
-            "Standard_NC24rs_v3",
-            "Standard_ND6s",
-            "Standard_ND12s",
-            "Standard_ND24s",
-            "Standard_ND24rs",
-            "Standard_NV6",
-            "Standard_NV12",
-            "Standard_NV24"
-          ],
-          "x-ms-enum": {
-            "name": "VirtualMachineSizeTypes",
-            "modelAsString": true
-          }
+          "description": "Specifies the size of the virtual machine. For more information about virtual machine sizes, see [Sizes for virtual machines](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-sizes?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json). <br><br> The available VM sizes depend on region and availability set. For a list of available sizes use these APIs:  <br><br> [List all available virtual machine sizes in an availability set](https://docs.microsoft.com/rest/api/compute/availabilitysets/listavailablesizes) <br><br> [List all available virtual machine sizes in a region](https://docs.microsoft.com/rest/api/compute/virtualmachinesizes/list) <br><br> [List all available virtual machine sizes for resizing](https://docs.microsoft.com/rest/api/compute/virtualmachines/listavailablesizes)"
```

Comment:
We can't remove the size string constants from 2020-06-01 swagger spec as this will considered as breaking change, however we should update the description with the updated doc links.

Reply:
Reverted.